.github/workflows/docs.yml

```diff
@@ -32,6 +32,7 @@ on:
       - 'ci/scripts/js_build.sh'
       - 'ci/scripts/python_build.sh'
       - 'ci/scripts/r_build.sh'
+      - 'docs/**'
```

Comment:
This should only trigger the "light" job and not the "complete" job?

Reply:
In order to do that we need to split the workflow file into two.

Reply:
Split into two workflows so we trigger the light build only on pull requests and the complete docs build only on pushes.